src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SingleField.jsx

```diff
@@ -1,46 +1,59 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect, useReducer} from 'react';
 import * as PropTypes from 'prop-types';
 import {compose} from '~/utils';
 import {connect, FastField} from 'formik';
 import {FieldPropTypes} from '~/FormDefinitions';
 
+const reducer = (state, action) => {
+    switch (action.type) {
+        case 'set':
+            return action.data;
+        case 'unMount':
+            return action.onChange(state, undefined);
+        default:
+            throw new Error();
+    }
+};
+
 export const SingleFieldCmp = ({inputContext, field, onChange, formik}) => {
     const FieldComponent = inputContext.fieldComponent;
     const [isInit, setInit] = useState(false);
 
+    const [previousValue, dispatch] = useReducer(reducer, undefined);
+
+    useEffect(() => {
+        return () => dispatch({type: 'unMount', onChange});
+    }, []);
```

Comment:
same here, we can add parameter field.value or something else to re trigger the useEffect

Reply:
No, for the same reason